docs/tasks/debug-application-cluster/audit.md

```diff
@@ -255,44 +300,55 @@ Events are POSTed as a JSON serialized `EventList`. An example payload:
 
 ```json
 {
-  "kind": "EventList",
-  "apiVersion": "audit.k8s.io/v1alpha1",
-  "items": [
-    {
-      "metadata": {
-        "creationTimestamp": null
-      },
-      "level": "Metadata",
-      "timestamp": "2017-06-15T23:07:40Z",
-      "auditID": "4faf711a-9094-400f-a876-d9188ceda548",
-      "stage": "ResponseComplete",
-      "requestURI": "/apis/rbac.authorization.k8s.io/v1beta1/namespaces/kube-public/rolebindings/system:controller:bootstrap-signer",
-      "verb": "get",
-      "user": {
-        "username": "system:apiserver",
-        "uid": "97a62906-e4d7-4048-8eda-4f0fb6ff8f1e",
-        "groups": [
-          "system:masters"
-        ]
-      },
-      "sourceIPs": [
-        "127.0.0.1"
-      ],
-      "objectRef": {
-        "resource": "rolebindings",
-        "namespace": "kube-public",
-        "name": "system:controller:bootstrap-signer",
-        "apiVersion": "rbac.authorization.k8s.io/v1beta1"
-      },
-      "responseStatus": {
-        "metadata": {},
-        "code": 200
-      }
-    }
-  ]
+    "apiVersion": "audit.k8s.io/v1beta1",
+    "items": [
+        {
+            "auditID": "24f30caf-d7d4-45d5-b7bd-e7af300d7886",
+            "level": "Metadata",
+            "metadata": {
+                "creationTimestamp": null
+            },
+            "objectRef": {
+                "apiGroup": "rbac.authorization.k8s.io",
+                "apiVersion": "v1",
+                "name": "jane",
+                "namespace": "default",
+                "resource": "roles"
+            },
+            "requestURI": "/apis/rbac.authorization.k8s.io/v1/namespaces/default/roles/jane",
+            "responseStatus": {
+                "code": 200,
+                "metadata": {}
+            },
+            "sourceIPs": [
+                "172.16.116.128"
+            ],
+            "stage": "ResponseComplete",
+            "timestamp": "2017-09-05T10:20:24Z",
+            "user": {
+                "groups": [
+                    "system:masters",
+                    "system:authenticated"
+                ],
+                "username": "kubecfg"
+            },
+            "verb": "get"
+        }
+    ],
+    "kind": "EventList",
+    "metadata": {}
 }
 ```
 
+### Audit-Id
+
+Audit-Id is a unique ID for each http request to kube-apiserver. The ID of events will be the
+same if they were generated from the same request. Starting from Kubernetes 1.8, if an audit
+event is generated for the request, kube-apiserver will respond the the Audit-Id in http header.
+We can see Audit-Id in debug info with command like `kubectl get pods --v=8`. Note that for some
```

Comment:
`We can see Audit-Id in debug info with command like `kubectl get pods --v=8`.`

Do we really want this level of details documented?

Reply:
Removed.